metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
I could be swayed to `surface_based_cape_cin`, especially with tab completion, I'm just thinking how people discuss it - SB-CAPE would be a common thing to say in our small circles.